tests/playwright/pages/wp-admin-page.js

```diff
@@ -73,15 +74,67 @@ module.exports = class WpAdminPage extends BasePage {
 	}
 
 	/**
-	 * Determine which experiments are active / inactive.
+	 * Activate / Deactivate Experiments.
 	 *
 	 * TODO: The testing environment isn't clean between tests - Use with caution!
 	 *
 	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 * @param {string} mode        - cli (default) or ui
 	 *
 	 * @return {Promise<void>}
 	 */
-	async setExperiments( experiments = {} ) {
+	async setExperiments( experiments = {}, mode = 'cli' ) {
```

Comment:
do we need this aggregated function?

Reply:
Right now our team want to test enabling / disabling experiments using the UI at least once to establish that the Experiments UI we've coded for each Widget works and keeps working.  This may be replaced by a Unit test in the future. 

I've kept this functionality optional in this aggregated function as a result.